src/main/resources/public/metrics.html

```diff
@@ -0,0 +1,159 @@
+<!--
```

Comment:
This is great for now. I think we need to also add this into the `Admin` bar on `navbar.html` so it can be accessed easily.

Reply:
Nevermind. I see its there.